pandas/core/indexes/api.py

```diff
@@ -1,23 +1,21 @@
 import textwrap
 import warnings
 
-from pandas.core.indexes.base import (Index,
-                                      _new_Index,
-                                      ensure_index,
-                                      ensure_index_from_sequences,
-                                      InvalidIndexError)  # noqa
+from pandas._libs import NaT, lib
+
+import pandas.core.common as com
+from pandas.core.indexes.base import (
+    Index, _new_Index, ensure_index, ensure_index_from_sequences)
+from pandas.core.indexes.base import InvalidIndexError  # noqa
```

Comment:
Her and elsewhere can you make the “noqa” specific?  I think it’s “noqa: F401”

Reply:
Sure makes sense! Thanks